Moblin/Various/Settings/SettingsStream.swift

```diff
@@ -831,6 +831,8 @@ class SettingsStream: Codable, Identifiable, Equatable, ObservableObject, Named
     @Published var kickSendMessagesTo: Bool = true
     var kickChatAlerts: SettingsKickAlerts = .init()
     var kickToastAlerts: SettingsKickAlerts = .init()
+    @Published var dliveStreamerUsername: String = ""
+    @Published var dliveLoggedIn: Bool = false
```

Comment:
we are not logged in. maybe replace by enabled toggle, or just remove it as I don't think other platforms have an enabled toggle